src/execution/ast/context.cpp

```diff
@@ -30,6 +30,16 @@
 // #include "execution/util/csv_reader.h" Fix later.
 #include "execution/util/execution_common.h"
 
+namespace {
+
+terrier::execution::ast::Field CreatePaddingElement(uint32_t id, uint32_t size, terrier::execution::ast::Context *ctx) {
+  terrier::execution::ast::Identifier name = ctx->GetIdentifier("__pad$" + std::to_string(id) + "$");
+  auto *pad_type = terrier::execution::ast::BuiltinType::Get(ctx, terrier::execution::ast::BuiltinType::Int8);
+  return terrier::execution::ast::Field(name, terrier::execution::ast::ArrayType::Get(size, pad_type));
```

Comment:
Move this into the `terrier::execution::ast` namespace directly above `StructType::Get()` where it's needed. Refactoring this way should clean it up.